scripts/lychee.js

```diff
@@ -23,6 +23,7 @@ lychee = {
 	justified				: true,		// use Flickr Justified Layout Like
 	image_overlay			: false,	// display Overlay like in Lightroom
 	image_overlay_default	: false,	// display Overlay like in Lightroom by default
+	image_overlay_type		: '',		// Overlay display type
```

Comment:
same principle as above: you need to have a
`image_overlay_type_default`.
What you are displaying in the settings is the default value.